site/layouts/shortcodes/orange-footer.html

```diff
@@ -154,7 +154,7 @@ <h3 class="accordion-header footer-heading" id="headingFour{{ $id }}">
     <ul class="navbar-nav gap-3 gap-md-4">
       <li><a class="nav-link gap-1" href="#"><svg width="1.875rem" height="1.875rem" focusable="false" aria-hidden="true"><use xlink:href="/docs/{{ .Site.Params.docs_version }}/assets/img/boosted-sprite.svg#locate"/></svg><span>Store locator</span></a></li>
       <li><a class="nav-link gap-1" href="#"><svg width="1.875rem" height="1.875rem" focusable="false" aria-hidden="true"><use xlink:href="/docs/{{ .Site.Params.docs_version }}/assets/img/boosted-sprite.svg#coverage"/></svg><span>Coverage checker</span></a></li>
-      <li><a class="nav-link gap-1" href="#"><svg width="1.875rem" height="1.875rem" focusable="false" aria-hidden="true"><use xlink:href="/docs/{{ .Site.Params.docs_version }}/assets/img/boosted-sprite.svg#contact"/></svg><span>Contact us</span></a></li>
+      <li>{{- if $in_examples }}<a class="nav-link gap-1" href="#">{{- else }}<a class="nav-link gap-1 active" href="#" aria-current="page">{{- end }}<svg width="1.875rem" height="1.875rem" focusable="false" aria-hidden="true"><use xlink:href="/docs/{{ .Site.Params.docs_version }}/assets/img/boosted-sprite.svg#contact"/></svg><span>Contact us</span></a></li>
```

Comment:
Instead of using the source for the naming, can't we use the purpose instead for better reusability? So instead of `in_examples`, use `show_footer_active_link` (or a better naming). So a new parameter.